charts/incubator/wyoming-piper/values.yaml

```diff
@@ -0,0 +1,43 @@
+image:
+  repository: docker.io/rhasspy/wyoming-piper
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:e62fa006b6fccda2f1be2f1fd8229bc6c1139b8adeb2e6d479f411e628bb1c90
```

Comment:
```suggestion
  repository: tccr.io/truecharts/wyoming-piper
  pullPolicy: IfNotPresent
  tag: v0.0.2@sha256:9768cc7435d14e3a79dc0fce566f0736adabcb494b579ca435c18fec3802871a
```

Reply:
Are we sure the digest is correct in the containers repo? It looks like the tag was updated to the new version but not the digest.

Reply:
It looks like it was automatically corrected: https://github.com/truecharts/containers/pull/26035